Lib/http/cookies.py

```diff
@@ -489,6 +489,9 @@ def __set(self, key, real_value, coded_value):
         dict.__setitem__(self, key, M)
 
     def __setitem__(self, key, value):
+        """Check if the key already exists, return without overwriting"""
```

Comment:
maybe with this change: 
```
class BaseCookie(dict):
    def __init__(self, input=None, replace_duplicates=True):
        """
        Initialize a BaseCookie instance.

        :param input: Initial cookie data.
        :param replace_duplicates: If True, new cookies replace existing ones with the same name.
                                   If False, the first cookie with a unique name is preserved.
        """
        self.replace_duplicates = replace_duplicates
        super().__init__()
        if input:
            self.load(input)

    def __setitem__(self, key, value):
        """
        Set a cookie.

        If replace_duplicates is False, an existing cookie with the same name will not be overwritten.
        """
        if not self.replace_duplicates and key in self:
            return
        super().__setitem__(key, value)
```
we could let end user to according common use or RFC?